common/containers/Tabs/Swap/components/wantToSwapMy.js

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import translate from 'translations';
+import {combineAndUpper} from 'api/bity';
+
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export default class WantToSwapMy extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    static propTypes = {
+        bityRates: PropTypes.any,
+        originAmount: PropTypes.any,
+        destinationAmount: PropTypes.any,
+        originKind: PropTypes.string,
+        destinationKind: PropTypes.string,
+        destinationKindOptions: PropTypes.array,
+        originKindOptions: PropTypes.array,
+        swapOriginKindTo: PropTypes.func,
+        swapDestinationKindTo: PropTypes.func,
+        swapOriginAmountTo: PropTypes.func,
+        swapDestinationAmountTo: PropTypes.func
+    };
+
+    onClickStartSwap() {
+
+    }
+
+    onChangeOriginAmount = (amount) => {
+        let originAmountAsNumber = parseFloat(amount);
+        if (originAmountAsNumber) {
```

Comment:
if they can get infinity into the input, I think they earned it :P

So... is this really a problem?